spring-cloud-generator/generate-all.sh

```diff
@@ -22,8 +30,20 @@ while IFS=, read -r library_name googleapis_location coordinates_version googlea
   echo "processing library $library_name"
   group_id=$(echo $coordinates_version | cut -f1 -d:)
   artifact_id=$(echo $coordinates_version | cut -f2 -d:)
-  bash $WORKING_DIR/generate-one.sh -c $library_name -i $artifact_id -g $group_id -p $PROJECT_VERSION -f $googleapis_location -x $googleapis_commitish
+  # setting pipefail allows to capture the exit code of a command called earlier
+  # in a pipe chain. This is necessary since piping to tee will always exit
+  # 0 if pipefail is not set
+  set -o pipefail
+  bash $WORKING_DIR/generate-one.sh \
+    -c $library_name \
+    -i $artifact_id \
+    -g $group_id \
+    -p $PROJECT_VERSION \
+    -f $googleapis_location \
+    -x $googleapis_commitish 2>&1 | tee tmp-generate-one-output || save_error_info $library_name
+  set +o pipefail
 done <<< $libraries
+rm tmp-generate-one-output
```

Comment:
I think this rm is redundant (and fails) because you already use `mv tmp-generate-one-output` instead of `cp tmp-generate-one-output` above
```suggestion
```